src/pybind/mgr/volumes/fs/async_job.py

```diff
@@ -244,6 +275,10 @@ def queue_job(self, volname):
                 self.jobs[volname] = []
             self.cv.notifyAll()
 
+        if subvolname is not None:
```

Comment:
Could you explain a bit about this handling? Its for async clones from the condition below, but, under what cases is `subvolname` None?

Reply:
I don't recall right now if I had seen crashes during testing due to absence of this line. It's been lot of time since I added it. I suppose I added it as a caution that `None` type doesn't get added to the queue.